index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
+                    <li>A "iat" claim may be included which contains information when the presentation was produced.</li>
+                    <li>A "iss" claim may be included which contains information on the presenter of the verifiable credentials. The "iss" claim MUST be one URI.</li>
+                </ul>
+            </ul>
+
+            <p>The following is a verifiable credential encoded as a JWT.</p>
+
+            <pre class="example nohighlight" title="Expressing a verifiable credential as a JWT (base64url-encoded)">
+eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJAY29udGV4dCI6WyJodHRwczovL3czLm9yZy8yMDE
+4L2NyZWRlbnRpYWxzL3YxIiwiaHR0cHM6Ly9leGFtcGxlLmNvbS9leGFtcGxlcy92MSJdLCJqdGkiOiJ
+1cm46dXVpZDozOTc4MzQ0Zi04NTk2LTRjM2EtYTk3OC04ZmNhYmEzOTAzYzUiLCJpc3MiOiJkaWQ6ZXh
+hbXBsZTplYmZlYjFmNzEyZWJjNmYxYzI3NmUxMmVjMjEiLCJhdWQiOiJodHRwczovL2Rtdi5leGFtcGx
+lLmdvdi9pc3N1ZXJzLzE0Iiwibm9uY2UiOiIzNDPCpyRGU0ZEYS0iLCJpYXQiOiIxNTQxNDkzNzI0Iiw
+iZXhwIjoiMTU3MzAyOTcyMyIsInR5cGUiOlsiVmVyaWZpYWJsZVByZXNlbnRhdGlvbiJdLCJ2ZXJpZml
+hYmxlQ3JlZGVudGlhbCI6WyJleUpoYkdjaU9pSlNVekkxTmlJc0luUjVjQ0k2SWtwWFZDSjkuZXlKQVk
+yOXVkR1Y0ZENJNld5Sm9kSFJ3Y3pvdkwzY3pMbTl5Wnk4eU1ERTRMMk55WldSbGJuUnBZV3h6TDNZeEl
+pd2lhSFIwY0hNNkx5OWxlR0Z0Y0d4bExtTnZiUzlsZUdGdGNHeGxjeTkyTVNKZExDSnpkV0lpT2lKa2F
+XUTZaWGhoYlhCc1pUcGxZbVpsWWpGbU56RXlaV0pqTm1ZeFl6STNObVV4TW1Wak1qRWlMQ0pxZEdraU9
+pSm9kSFJ3T2k4dlpHMTJMbVY0WVcxd2JHVXVaMjkyTDJOeVpXUmxiblJwWVd4ekx6TTNNeklpTENKcGM
+zTWlPaUpvZEhSd2N6b3ZMMlJ0ZGk1bGVHRnRjR3hsTG1kdmRpOXBjM04xWlhKekx6RTBJaXdpYVdGMEl
+qb2lNVFUwTVRRNU16Y3lOQ0lzSW1WNGNDSTZJakUxTnpNd01qazNNak1pTENKMGVYQmxJanBiSWxabGN
+tbG1hV0ZpYkdWRGNtVmtaVzUwYVdGc0lpd2lVSEp2YjJaUFprRm5aVU55WldSbGJuUnBZV3dpWFN3aVk
+yeGhhVzBpT25zaVlXZGxUM1psY2lJNk1qRjlmUS5UUXltWms2Z0IyekRhUmVFNlVUYTlLQnA2MVRlX21
+0bzl3V1JDZDFDcWNkQWpOMFNZMTItVWhPbHN4em4tWGhaRE50R2RUd3pZX1ZVUklPR2JiQkROZ25HYlR
+zdHJ3bjgtU29oX3FpZzJsc3k3ZlM1ckF6QmdtZGFmSkxiYUtTQ0VsSGJ5RElsMVZ4NFFmYzk0LVdOVWJ
+pQ1RTaEp3dENJN204MmxMWjltSmdmIl19.SR-zmsg-i8n9WNj43zYX0LERb8UWfgYyHFLG3R2HmbhAS5
+mLFsy3xA1D8f6ksoT5UxOwr5T7uuj1UkC8v3aX1Z6yZus1HhhdhuhPnsMvkbBLJl7G6u4z8Fq7AJxT6e
+BhndD4Q6TfhcWXisrZaR1Al1921L_CfOqq5ZD1C1EdaoM
+          </pre>
+
+            <p>The example above contains the following JWT header and payload.</p>
+            <pre class="example nohighlight" title="Presentation JWT header">
+{
+  "alg": "RS256",
+  "typ": "JWT"
+}
+          </pre>
+
+            <pre class="example nohighlight" title="Presentation JWT payload">
+{
+  "@context": [
+    "https://w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "jti": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
+  "iss": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+  "aud": "https://dmv.example.gov/issuers/14",
+  "nonce": "343§$FSFDa-",
+  "iat": "1541493724",
+  "exp": "1573029723",
+  "type": ["VerifiablePresentation"],
+  "verifiableCredential": ["
```

Comment:
This should probably change to `verifiableCredentialJwt` as the value is encoded.

Reply:
I updated the the JWT data model to reflect the outcome of our JWT discussion. The changes should address this issue.